### PR TITLE
Creating an apple website clone

### PR DESCRIPTION
**Pull request by alvinlouis29**
the aim of this project is to create a clone of an apple old website of 2014 from http://archive.fo/UW4oR using CSS and Html

### CODE REVIEW

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @alvinlouis29        
#### Great Job so far, but there are a few more things required. 
### General
- [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. 
- [ ] The screenshot in the readme file is broken,lets fix that.
#### Navbar
* [ ] The Search field on the rightmost section should look similar to the original

#### Navigation background
* [ ] It uses CSS [`background -> gradient`](https://www.w3schools.com/cssref/css3_pr_background.asp) for background
### Original 


### Yours


- [ ] This Four elements should look like the original
### Original 

### Yours

#### Footer
- [ ] Kindly note the footer should look similar to the original
### Original 


### Yours

- [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout)

### Make changes and submit another code review request.
*Happy coding*

**Comment by majovanilla** [comment]
### Code review under progress by Maria Reyes

**Comment by majovanilla** [comment]
Hey @alvinlouis29 , 👋

## You did a *great* job implementing the required changes on this project! 💜 

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Always remember to check your HTML using the [validator](https://validator.w3.org/). You have one error. Please fix it.
Screenshot from 2020-02-28 20-48-36

- Improve your Readme file. It should be descriptive of the project. You are using the right template! Just fill it up with the information about the great work you did here. And fix the screenshot as requested by @kobiyoyo , please!!

-Please write meaningful pull request messages. Check this [example](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

- The search bar is not quite there yet. Please change the background so that it matches the original one. It should include the search icon, too. And it should have a rounded shape.

Your search bar:
Screenshot from 2020-02-28 20-52-55

Original search bar:
Screenshot from 2020-02-28 20-53-25

- You are using buttons on your navbar. That is not the right use of buttons (check [here](https://stackoverflow.com/questions/21715162/when-should-i-use-a-button-button-or-a-link-a-in-html) and [here](https://developer.mozilla.org/en/docs/Web/HTML/Element/button) for more on that. If you check the original page, the navbar has links `<a>` tags. Please change this.


### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- You can add the hover effects to the text on the main image and the links on the footer.


## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes



You can find me on Social Media below 👇 . Remember to say hi!

[GitHub](https://github.com/majovanilla) [Twitter](https://twitter.com/MajoVanilla) [LinkedIn](https://www.linkedin.com/in/majoreyesparroquin/)

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Hi, @alvinlouis29 great job so far :clap: :+1: :100:, but :warning: I have seen that you skipped several requirements by the previous TSE :arrow_backward:, please return back and make them valid also align the apple logo inside the navbar it's not aligned correctly, and fix your stickler errors.

### Please make all the required changes before you request another code review.
## Happy Coding :tophat: 
_If you feel like stuck or need help don't hesitate to contact me on slack_

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## Impressive progress so far 💪! Thanks for making all the changes 👏.

### However, we need to work on some changes before we can merge.

- There is no live demo link in your project README. Please add that. Also, the screenshot is not displaying.
- The search icon in your navbar is not aligned properly. Please fix that to match the original.

   image

Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻 

-------------------------------------------------------------------------------------------------------------------

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                </svg>
+              </div>
+                
+                
+            </li>
+      
+            
+            </ul>
+        </nav>
+        <section class="main-content">
```
Comment (voscarmv) [inline index.html:54]: Please use a proper `<main>` semantic tag here, instead of just a `<section>` tag.

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work! 🖥️ 👀 ✨ 
- Good job implementing the requested changes.
- Nice `README` file.

## However, some changes are still required... 🧰 🔨 🔧 

- Please fix the following HTML error found using the [Nu HTML validator](https://validator.w3.org/nu/#textarea)
image

- Please use a proper `<main>` semantic tag, as indicated on the code comments.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Comment by bertil291utn** [comment]
**Status:** Approved ✅
# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

 ## Ship it

image

Please merge to master branch and move on with the next assignment.

You're doing  a great job keep going 💻 🚀 🔥

**[TAKE IN MIND]**
Next projects:

- [ ] Be careful with README file, should be more professional
- [ ] Your commit messages are great, however your box message pull request should be like so: title follow by a description of what have you done.

Check again [this lesson](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) to improve these requirements



> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Pull request merged**